bftengine/src/preprocessor/tests/preprocessor_test.cpp

```diff
@@ -435,6 +441,7 @@ TEST(requestPreprocessingState_test, requestTimedOut) {
   usleep(replicaConfig.preExecReqStatusCheckTimerMillisec * 1000);
   timers.evaluate();
   ConcordAssert(preProcessor.getOngoingReqIdForClient(clientId) == 0);
+  clearDiagnosticsHandlers();
```

Comment:
Thank you for including that and handling those changes in the pre-processor.
The reason why I have included the try-catch is exactly that the tests are trying to register handlers that have already been registered.